transport_nantes/topicblog/templatetags/launcher.py

```diff
@@ -13,3 +13,15 @@ def launcher(slug):
     if launcher is None:
         logger.error(f"item_teaser failed to find slug: \"{slug}\".")
     return {'launcher': launcher}
+
+@register.inclusion_tag('topicblog/template_tags/item_teaser.html')
+def item_teaser(slug):
+    launcher = TopicBlogLauncher.objects.filter(
+        slug__iexact=slug, publication_date__isnull=False).first()
+    if launcher is None:
+        logger.error(f"item_teaser failed to find slug: \"{slug}\".")
```

Comment:
Good to log the error, but then falling through to the next line will fail violently, no?